### ISSUE TITLE
[BUG] some videos do not show subtitles

### ISSUE THREAD

**Issue by milindgoel15** [anilbeesetti/nextplayer#727]
**Describe the bug**
Though I can't seem to figure out which type of videos are like this, but some of the videos/movies that I download do not show any subtitles in the video player when they actually have one included. 

I confirmed it via VLC player.

**To Reproduce**
Not sure but

1. Download a movie torrent with included subtitles.
2. 50% chance will be the subtitle won't show up in the subtitle option

**Expected behavior**
Subtitles show up

**Screenshots**
![Screenshot_2023-11-10-10-58-26-276_dev.anilbeesetti.nextplayer.jpg](https://github.com/anilbeesetti/nextplayer/assets/45682747/01111317-cc7c-49cc-be6d-5b5c5a24fb6a)

![Screenshot_2023-11-10-10-58-40-300_org.videolan.vlc.jpg](https://github.com/anilbeesetti/nextplayer/assets/45682747/1cd220af-5ab3-4187-ba72-46d24b2c8319)

**Device info (please complete the following information):**
 - Device: Xiaomi 11 lite ne 5g
 - Android version: 13
 - App version: 0.10.0

**Additional context**
NA

**Comment by anilbeesetti** [comment]
@milindgoel15 can you send the specific video file or torrent

**Comment by milindgoel15** [comment]
> @milindgoel15 can you send the specific video file or torrent

Shall I send the link to the torrent on other platforms? Are you on telegram?

**Comment by anilbeesetti** [comment]
Yeah I'm on telegram but can you send the link through pastebin.com

**Comment by milindgoel15** [comment]
> Yeah I'm on telegram but can you send the link through pastebin.com

Can you share your telegram username?

**Comment by anilbeesetti** [comment]
@milindgoel15 https://t.me/user_c3po

**Comment by anilbeesetti** [comment]
@milindgoel15 The video file provided by you contains subtitles in DVD format which doesn't supported by exoplayer natively yet. Although i can try using ffmpeg but it's gonna take some time

**Comment by milindgoel15** [comment]
Cool

**Comment by drogga** [comment]
So .sub & .idx and not .vtt, .ssa/.ass, .vob or other formats ?
_...I wonder if the other Open Sourced players (some listen in the ReadMe) support those..._

**Comment by uwuMocha** [comment]
I am not sure if it's appropriate to submit here since this does not seem specifically related to being DVD subtitle format, but I'm having a similar issue. 

I've included a paste of the mediainfo subtitles section below. The player still detects subtitles for both files in the subtitles selector, they both have ASS format, but for one of the files the subs do not show up at all on screen while playing the video. The files that do work are any subsplease files, the file in question that does not work is from the Kaleido-Subs releases of Blue Archive The Animation, none of those work. I suspect this could be related to the kaleido-subs size/bitrate/framerate of the subtitle or the fact that kaleido shows "Muxing mode: zlib" in one line of the mediainfo.
---------------------------------
--Subsplease-- works
---------------------------------
Text
ID                                       : 3
Format                                   : ASS
Codec ID                                 : S_TEXT/ASS
Codec ID/Info                            : Advanced Sub Station Alpha
Duration                                 : 2 min 42 s
Bit rate                                 : 147 b/s
Frame rate                               : 0.283 FPS
Count of elements                        : 46
Compression mode                         : Lossless
Stream size                              : 2.94 KiB (0%)
Title                                    : English subs
Language                                 : English
Default                                  : Yes
Forced                                   : No
---------------------------------
--Kaleido-Subs-- Doesn't work
---------------------------------
Text #1
ID                                       : 3
Format                                   : ASS
Muxing mode                              : zlib
Codec ID                                 : S_TEXT/ASS
Codec ID/Info                            : Advanced Sub Station Alpha
Duration                                 : 22 min 13 s
Bit rate                                 : 121 kb/s
Frame rate                               : 43.188 FPS
Count of elements                        : 57587
Compression mode                         : Lossless
Stream size                              : 19.2 MiB (1%)
Title                                    : Full Subtitles [Kaleido-subs]
Language                                 : English
Default                                  : Yes
Forced                                   : No

Thanks for working on this project, it is the best player on Android right now in my opinion, and again I'm sorry if I should have made a full new post for this.
